src/matrix/storage/idb/schema.ts

```diff
@@ -86,13 +92,14 @@ function createE2EEStores(db) {
 }
 
 // v5
-async function migrateEncryptionFlag(db, txn) {
+async function migrateEncryptionFlag(db: IDBDatabase, txn: IDBTransaction): Promise<void> {
     // migrate room summary isEncrypted -> encryption prop
     const roomSummary = txn.objectStore("roomSummary");
     const roomState = txn.objectStore("roomState");
-    const summaries = [];
-    await iterateCursor(roomSummary.openCursor(), summary => {
+    const summaries: any[] = [];
```

Comment:
reason to not use `SummaryData` here is because we're dealing with the `isEncrypted` that isn't supported anymore?

Reply:
I mentioned in the comment that I would just leave the "old" entries unspecified (doesn't seem all that valuable). In this case it's possible to add the `& { isDisabled: boolean }` annotation, and work that out.

In addition, `SummaryData` is currently from a JS module, and this implicily typed as `any`.